src/realm/__tests__/realmReducer-test.js

```diff
@@ -25,7 +25,6 @@ describe('realmReducer', () => {
         user_id: undefined,
         isAdmin: false,
         twentyFourHourTime: false,
-        jitsiServerUrl: null,
```

Comment:
Hmm, this doesn't seem to be quite enough to update the tests -- see the results of running `tools/test`.

Reply:
Hmm, I'd been running the test suite locally so I'm not sure how this slipped through, but I've updated to fix the test.